Dockerfile

```diff
@@ -1,9 +1,9 @@
 # syntax=docker/dockerfile:1
 
-ARG QBITTORRENT_VERSION=5.0.3
+ARG QBITTORRENT_VERSION=5.1.2
```

Comment:
Latest seems to be 5.1.4 https://github.com/qbittorrent/qBittorrent/releases/tag/release-5.1.4